apiary.apib

```diff
@@ -3400,6 +3404,9 @@ Update a draft invoice. Booked invoices cannot be updated.
         + expected_payment_method (ExpectedPaymentMethod, optional, nullable)
         + custom_fields (array[CustomFieldValue], optional)
         + document_template_id: `179e1564-493b-4305-8c54-a34fc80920fc` (string, optional)
+        + invoice_currency (object, optional)
```

Comment:
Q: Isn't the `invoice` prefix redundant since we are already on an invoice resource?

I think solely `currency` or `price_currency` might be better?

@Miljar what are your thoughts?

```suggestion
        + currency (object, optional)
```